numpy/_core/_add_newdocs_scalars.py

```diff
@@ -369,6 +369,24 @@ def add_newdoc_for_scalar_type(obj, fixed_aliases, doc):
         False
         """))
 
+    add_newdoc('numpy._core.numerictypes', float_name, ('__trunc__',
+        f"""
+        {float_name}.__trunc__() -> int
+
+        Return the floating point number with the fractional part removed,
+        leaving the integer part.
+
+        .. versionadded:: 2.3
```

Comment:
```pycon
>>> import numpy as np
>>> np.__version__
'2.2.5'
>>> np.float64.__trunc__
<method '__trunc__' of 'float' objects>
```

so technically speaking, `float64.__trunc__` *wasn't* added in 2.3

Reply:
Ah good catch. Let me throw it under an `if`

Reply:
added a somewhat hacky `if`, let me know if this is ok.